src/lib/features/project/project-flag-creators-read-model.ts

```diff
@@ -0,0 +1,31 @@
+import type { Db } from '../../db/db';
+import type { IProjectFlagCreatorsReadModel } from './project-flag-creators-read-model.type';
+
+export class ProjectFlagCreatorsReadModel
+    implements IProjectFlagCreatorsReadModel
+{
+    private db: Db;
+
+    constructor(db: Db) {
+        this.db = db;
+    }
+
+    async getFlagCreators(
+        project: string,
+    ): Promise<Array<{ id: number; name: string }>> {
+        const result = await this.db('users')
+            .distinct('users.id')
+            .join('features', 'users.id', '=', 'features.created_by_user_id')
+            .where('features.project', project)
+            .select([
+                'users.id',
+                'users.name',
+                'users.username',
+                'users.email',
+            ]);
+        return result.map((row) => ({
+            id: Number(row.id),
+            name: String(row.name || row.username || row.email),
```

Comment:
Don't we have a standardized function to extract this from users? Do we handle it differently here than in other places?

Reply:
I think we do it in a few places. I'm not sure how to generalize it since query can give those fields different names when you use aliases.